lib/nodes/div.js

```diff
@@ -2,9 +2,15 @@ const PREFIX = '\n';
 const SUFFIX = '';
 
 module.exports = (props, children) => {
+    // If a node has children: For each string node, add a prefix and suffix.
+    if (children.length > 0) {
+        return children.map(node => (typeof node === 'string' ? (PREFIX + node + SUFFIX) : node));
```

Comment:
This line seems pretty long. Have you ran it through the linter/prettier? I'm not able to check out the branch at the moment to check.

Other than that, looks good.